### DISCUSSION TITLE
SFRA - How to enable sourcemaps in Sass

### DISCUSSION THREAD

**Discussion by ahelja** [SalesforceCommerceCloud/sfcc-ci#406] [Team Posts]
Is there an easy way to enable the scss mappings inside webpack.config?
I tried devtool: 'source-map', but no luck. 🤔

**Comment by dlferro** [comment]
You'll want to make sure you're leveraging the sourceMap option in the loader. See example in the documentation below:

https://github.com/webpack-contrib/sass-loader
